homeassistant/components/homekit/accessories.py

```diff
@@ -302,13 +302,6 @@ def available(self):
     async def run(self):
         """Handle accessory driver started event.
 
-        Run inside the HAP-python event loop.
-        """
-        self.hass.add_job(self.run_handler)
```

Comment:
Oh. We shouldn't have called `add_job` from an async context. Guess `call_soon_threadsafe` works fine from an async context.